Cargo.lock

```diff
@@ -1184,7 +1184,7 @@ dependencies = [
  "parking_lot 0.10.2 (registry+https://github.com/rust-lang/crates.io-index)",
  "wayland-client 0.23.6 (registry+https://github.com/rust-lang/crates.io-index)",
  "winapi 0.3.8 (registry+https://github.com/rust-lang/crates.io-index)",
- "winit 0.22.2 (git+https://github.com/michaelkirk/winit?branch=mkirk/fix-stdweb-dpi)",
+ "winit 0.22.2 (git+https://github.com/rust-windowing/winit)",
```

Comment:
They've also since merged my earlier PR: "DPI vs. custom cursor icon: https://github.com/rust-windowing/winit/pull/1652", so I'm referencing the official `rust-windowing/winit` instead of my fork.